src/ansible_navigator/ui_framework/utils.py

```diff
@@ -12,7 +12,7 @@ def convert_percentage(dyct: dict, keys: List, pbar_width: int) -> None:
     not recursive
     80% = 80%|XXXXXXXX  |
 
-    :pararm dicts: a list of dictionaries
+    :param dicts: a list of dictionaries
     :type dicts: list of dictionaries
```

Comment:
Looks like this is what is causing the problem in sphinx.
```suggestion
    :type dicts: list[dict]
```